provers/sp1/driver/src/lib.rs

```diff
@@ -203,41 +169,44 @@ impl Prover for Sp1Prover {
 }
 
 fn init_verifier() -> Result<PathBuf, ProverError> {
```

Comment:
you don't have to, this is to check whether your verifier.sol files are outdated. Since the contracts in `prover/sp1/contracts` are committed anyways so you can ignore this. The reason for the existence is to check updates if the contract version is not aligned with ur circuit version